modules/caddyhttp/reverseproxy/reverseproxy.go

```diff
@@ -256,8 +256,15 @@ func (h *Handler) Provision(ctx caddy.Context) error {
 			}
 
 			h.HealthChecks.Active.httpClient = &http.Client{
-				Timeout:   timeout,
-				Transport: h.Transport,
+				Timeout: timeout,
+			}
+
+			if h.TransportRaw != nil {
+				mod, err := ctx.LoadModule(h, "TransportRaw")
+				if err != nil {
+					return fmt.Errorf("loading transport: %v", err)
```

Comment:
I hope you don't mind my looking at the fix for the issue I submitted. 😊

Consider using `%w` to wrap the error instead, or is that a newer feature than the version of Go you're targeting?

Reply:
Thanks for reviewing the PR!

I'd rather we only use `%w` if we have a deliberate reason or need to expose the underlying error; in an application module such as this one we find that this is very uncommon (it's more common in libraries).

Reply:
Ah, right, I didn't think of that. Good point.